_inc/client/components/jumpstart/index.jsx

```diff
@@ -60,15 +60,12 @@ class JumpStart extends Component {
 							{ __( 'Your Jetpack site is ready to go!' ) }
 						</h3>
 						<p className="jp-jumpstart-card__description-text">
-							{ __(
-								'We’re now collecting stats, securing your site, and enhancing your editing experience. ' +
-									'Pretty soon you’ll be able to see everything going on with your site right through Jetpack! Welcome aboard.'
-							) }
+							{ __( 'We’re now collecting stats and securing your site. Welcome aboard.' ) }
 						</p>
 						<p className="jp-jumpstart-card__description-text">
 							{ __(
-								'Activate Jetpack’s recommended features to get the most out of your site. ' +
-									'Don’t worry, features can be activated and deactivated at any time. ' +
+								'Next, activate Jetpack’s recommended feature set to maximize your site’s security and performance. ' +
+									'Learn more about what’s included. ' +
```

Comment:
Sorry I induced you in error, @eliorivero — this sentence should replace the text in the link below, like so:

> ### Your Jetpack site is ready to go!
>
> We’re now collecting stats and securing your site. Welcome aboard.
>
> Next, activate Jetpack’s recommended feature set to maximize your site’s security and performance. (Learn more about what’s included.)
>
> **[Activate recommended features]**
>
> (Skip, and explore features individually.)

I've fixed the copy (not the mock) in the original issue: https://github.com/Automattic/jetpack/issues/11534

Reply:
Updated!